jenkins/scripts/bare_metal_lab/default_vars/vars.yaml

```diff
@@ -28,3 +28,7 @@ bare_metal_hosts:
 
 EPHEMERAL_CLUSTER: "minikube"
 EXTERNAL_VLAN_ID: 3
+# In order to run multiple instances (one per jump host), we list the known
+# hosts and tell DNSMasq to ignore all other.
+DHCP_HOSTS: "{{ bare_metal_hosts | map(attribute='mac') | join(';') }}"
```

Comment:
In `bare_metal_hosts` we have to macs do these include the default jump host or where the ephemeral test for the CI would be running

Reply:
The jump hosts are not included. This is just the macs of the hosts used for the target cluster.

Reply:
so the sever that host the CI ephemeral cluster will never become a node

Reply:
Exactly